### ISSUE TITLE
Index out of range error

### ISSUE THREAD

**Issue by ian-r-rose** [CityOfLosAngeles/count-machine#21]
Trying to run this on a sample video generates the following error:
```pytb
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/ian/anaconda3/envs/count-machine/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/ian/anaconda3/envs/count-machine/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/c/Users/itacid457/repos/count-machine/GUI/controller/main_controller.py", line 66, in update_video_canvas
    object_classes,color_table = self.get_selected_objects_list()
  File "/mnt/c/Users/itacid457/repos/count-machine/GUI/controller/main_controller.py", line 102, in get_selected_objects_list
    color_rgb = (color_bgr[2], color_bgr[1], color_bgr[0])
IndexError: string index out of range
```
and then nothing seems to run.

To reproduce, I did the following:
1. Load a sample video
1. Check "Car" in the "Valid Objects" selector
1. Press "Generate"

cc @vahedi25411

**Comment by vahedi25411** [comment]
The error fixed on [this](https://github.com/CityOfLosAngeles/count-machine/commit/becb69ea26c02f3f23292e25df16c2906661d75e) commit

**Comment by ian-r-rose** [comment]
Fixed by becb69ea26c02f3f23292e25df16c2906661d75e
